.gitignore

```diff
@@ -1,7 +1,9 @@
 _ReSharper*
 bin/
 obj/
-.svn/
 *.user
 *.suo
 libraries/
+# Visual Studio 2015/2017 cache/options directory
+.vs/
+publish/

```

Comment:
Any other `.gitignore`s from `source/.gitignore` that should be brought over?

Reply:
There was a whole bunch of stuff in the second .gitignore file, some of it looked really dated/legacy e.g. exclusion of files for svn repos, specific products. I think it should be fine if we keep it as simple as possible and only ignore what really needs to be ignored.